crates/bevy_ecs/src/world/mod.rs

```diff
@@ -1335,6 +1359,15 @@ impl World {
         unsafe { self.as_unsafe_world_cell_readonly().get_resource() }
     }
 
+    /// Gets a reference to the resource of the given type if it exists
```

Comment:
This is the same doc as is on `get_resource`. We should make the distinction clear between the two.

```suggestion
    /// Gets a reference including change detection to the resource of the given type if it exists.
```